karma.conf.js

```diff
@@ -55,7 +55,7 @@ module.exports = (config) => {
                 thresholds: {
                     global: {
                         statements: 97,
-                        branches: 90,
+                        branches: 92,
```

Comment:
🙏

Reply:
If you like this, you're gonna love: https://github.com/adobe/spectrum-web-components/pull/728/files#diff-a2a3b7b0c9c3b4b93b4aebf4e3ec3cfbR57